src/mixins/utils.ts

```diff
@@ -23,6 +24,50 @@ export default class UtilsMixin extends Vue {
     }
   }
 
+  get pageIcon () {
```

Comment:
I think this might be better off in the Globals.ts - that way it's not mixed in to every page that may not require it.

Reply:
You're probably right there, the only reason I put it in utils.ts is because that's where the title code is aswell. I'll move the icon code when I get some time/after you push that change. One one hand would it be a good idea for the title code to be in globals aswell? I suppose it is a lot simpler though so maybe it's fine where it is. Thanks for taking a look.

Reply:
Yah - honestly i need to pull a lot of stuff from the util mixin. :)

I had another thought too, can you update so that the color of the ring inherits the primary color theme? This way once I implement more themes, the favicon will replicate the ring color.

You can prolly grab this somewhere from `Vue.$vuetify..` or `this.$vuetify` depending where you're calling from.